lib/absinthe/schema/notation/sdl_render.ex

```diff
@@ -368,12 +368,11 @@ defmodule Absinthe.Schema.Notation.SDL.Render do
     empty()
   end
 
-  defp implements(interface_types) do
-    interface_names = Enum.map(interface_types, & &1.name)
-
+  defp implements(interface_names) do
     concat([
       " implements ",
-      join(interface_names, " & ")
+      Enum.map(interface_names, &(&1 |> Atom.to_string() |> String.capitalize()))
```

Comment:
I don't think this is always correct, we should have a way to access the `name` of the interfaces directly, right?

Reply:
I'm not sure, this is what the object looked like for the implementing object:
```
    %Absinthe.Blueprint.Schema.ObjectTypeDefinition{
          __private__: [__absinthe_referenced__: true],
          __reference__: %{
            location: %{
              file: "/Users/eotopalik/newrelic/absinthe/test/mix/tasks/absinthe.schema.sdl_test.exs",
              line: 99
            },
            module: Mix.Tasks.Absinthe.Schema.SdlTest.TestSchemaWithMods
          },
          description: nil,
          directives: [],
          errors: [],
          fields: [
            %Absinthe.Blueprint.Schema.FieldDefinition{
              __private__: [],
              __reference__: %{
                location: %{
                  file: "/Users/eotopalik/newrelic/absinthe/lib/absinthe/phase/schema/introspection.ex",
                  line: 74
                },
                module: Absinthe.Phase.Schema.Introspection
              },
              arguments: [],
              complexity: 0,
              config: nil,
              default_value: nil,
              deprecation: nil,
              description: "The name of the object type currently being queried.",
              directives: [],
              errors: [],
              flags: %{reserved_name: true},
              function_ref: nil,
              identifier: :__typename,
              middleware: [
                {:ref, Absinthe.Phase.Schema.Introspection, :typename}
              ],
              module: Absinthe.Phase.Schema.Introspection,
              name: "__typename",
              source_location: nil,
              triggers: %{},
              type: :string
            },
            %Absinthe.Blueprint.Schema.FieldDefinition{
              __private__: [],
              __reference__: %{
                location: %{
                  file: "/Users/eotopalik/newrelic/absinthe/test/mix/tasks/absinthe.schema.sdl_test.exs",
                  line: 101
                },
                module: Mix.Tasks.Absinthe.Schema.SdlTest.TestSchemaWithMods
              },
              arguments: [],
              complexity: {:ref,
               Mix.Tasks.Absinthe.Schema.SdlTest.TestSchemaWithMods,
               {Absinthe.Blueprint.Schema.FieldDefinition, {:human, :name}}},
              config: {:ref,
               Mix.Tasks.Absinthe.Schema.SdlTest.TestSchemaWithMods,
               {Absinthe.Blueprint.Schema.FieldDefinition, {:human, :name}}},
              default_value: nil,
              deprecation: nil,
              description: nil,
              directives: [],
              errors: [],
              flags: %{},
              function_ref: {:human, :name},
              identifier: :name,
              middleware: [
                {:ref, Mix.Tasks.Absinthe.Schema.SdlTest.TestSchemaWithMods,
                 {Absinthe.Blueprint.Schema.FieldDefinition, {:human, :name}}}
              ],
              module: Mix.Tasks.Absinthe.Schema.SdlTest.TestSchemaWithMods,
              name: "name",
              source_location: nil,
              triggers: {:ref,
               Mix.Tasks.Absinthe.Schema.SdlTest.TestSchemaWithMods,
               {Absinthe.Blueprint.Schema.FieldDefinition, {:human, :name}}},
              type: :string
            }
          ],
          flags: %{},
          identifier: :human,
          imports: [],
          interface_blueprints: [],
          interfaces: [:being],
          is_type_of: {:ref,
           Mix.Tasks.Absinthe.Schema.SdlTest.TestSchemaWithMods,
           {Absinthe.Blueprint.Schema.ObjectTypeDefinition, :human}},
          module: Mix.Tasks.Absinthe.Schema.SdlTest.TestSchemaWithMods,
          name: "Human",
          source_location: nil
    }
```

Reply:
is it possible that I'm missing something in my test absinthe schema?

Reply:
@emeryotopalik you need to use the `type_definitions` arg to the `defp render` function to go find the interfaces and use their names from those types.